route/test/test_route_generator.cpp

```diff
@@ -484,8 +486,8 @@ TEST(RouteGeneratorTest, test_set_active_route_cb)
     route::RouteGeneratorWorker worker2;
     worker2.georeference_cb(msg_ptr); 
     worker2.setWorldModelPtr(cmw);
-    worker2.set_route_file_path("../resource/route/");
-    worker2.pose_cb(mpt);
+    worker2.set_route_file_path("/workspaces/carma_ws/src/carma-platform/route/resource/route/");
```

Comment:
Is this normal to use an absolute path instead of relative? Might cause breakages on other machines right?

Reply:
No I tried changing it maybe the map can be loaded properly. which didn't help. revert back.